### ISSUE TITLE
AttachMembers not working in Fable REPL

### ISSUE THREAD

**Issue by Jmaharman** [fable-compiler/Fable#2383]
In a recent twitter exchange there was a discussion regarding Hotwire and Stimulus, where by it was [mentioned](https://twitter.com/Emmet_Ray/status/1360671637037793280?s=20) that they wondered whether Fable would support [static class features](https://github.com/tc39/proposal-static-class-features/), specifically mentioning targets.

I had a look at the API and mocked something up to see what the output would be, starting off in the REPL, however I soon ran into an issue where AttachMembers was not working.

Here is the code:

```fs
open Fable.Core
open Fable.Core.JsInterop
open Browser
open Browser.Types

// Stimulus controller abstract class
[<Erase;AttachMembers>]
type Controller() =
    abstract member initialize : unit -> unit
    default _.initialize() = ()

    abstract member connect : unit -> unit
    default _.connect() = ()

    abstract member disconnect : unit -> unit
    default _.disconnect() = ()

    member this.nameTarget : HTMLElement = failwith "JS only"

[<AttachMembers>]
type GreetController() =
    inherit Controller()

    static let targets = [| "name" |]

    override this.initialize() =
        console.log("Initializing")

    member this.greet() =
        let element = this.nameTarget :?> HTMLInputElement
        let name = element.value
        console.log((sprintf "Hello, %s!" name))
```

And this is what it should give me:

```js
export class GreetController extends Controller {
    constructor() {
        super();
    }
    static [".cctor"]() {
        const targets = ["name"];
    }
    initialize() {
        console.log(some("Initializing"));
    }
    greet() {
        const this$ = this;
        const element = this$.nameTarget;
        const name = element.value;
        console.log(some(toText(printf("Hello, %s!"))(name)));
    }
}
```

However the REPL says that AttachMembers is not defined.

The version information says:
REPL: 3.1.4
Fable: 3.1.4

I thought AttachMembers was added before this version, so I'm unsure why it can't resolve AttachMembers.

https://fable.io/repl/#?code=PYBwpgdgBAYghgIwDZgHQGFgCcwChSSyIobZoBSAzgJIQAuYWo+40AQkwO6WMuEfBujVABUAnuEq5cAehlQAynQCWAWwCuSdZSgBjYPSZIUWKIkp0scXXT1I4lKQG0APAFErPANwBBOnWsACwBZMFUERkoAPgBdXDoJMChMQ2BjRgAKAEooAF5cKELzS2tbVTCI02UIZRU4JGUALySALih1GtsAWij2zoLCgBMwADM4TVsAfVRq2uV6prBsvKhs6UKzBAsrGyhy8MY9AwgwXbaO2qgevtqBqGGxiahp-QgTm2Xc1az1oq2S3b7Sr3ZSUV7vWznTpXXoXOh3B7jJBTVCDUHg050T7fX57CqHOiBUGoCBwcoiOBYADmYEhUAAEiJggAZNwocr0FYdBDAAAeUAARALpK4-AFdCF8VhonEEuAoABxHC0lKWNImT4DaqBRiXVVGDU-AYWOAqXRQFC2ALU2k6L5OQWk8oCqBxAZAglEyioKnKrE5fIbC20qBgdmQWxfQnEp1gCk2yEAfl6jJZtBA6jobLCEbuwdssZWYZz9FQADd6uo8EG5FBaBYwHBBlBgCM9A4VBAqVAxMB1EdNM3BsAoISkhEkIIADRQBCZnt9gDkzcnPFHiSglDgI1pYjztctUELUa9JLJccpNLoiYrWmrG1elDSaEnVIyGUoICw1TobYF9LDScZwAUkoABCF1YyyH4gA&html=DwCwLgtgNgfAUHUBTAhgE3gAk8CSwqbhgAOAtEgI4CuAlgG4C8A5AMID2AdmEt2QCoBPEkmaYAxlx7cWPAB5gA9OGgBuCSBQAnAM75G1MADMyADmZYcO8VtokwmHVvGMARMRI6AXIsXVOJADWAOYAdJIQigACAO5IAEYRJFy8YDqK4tQ6YOwQFFBIeNw6UQCMoQBMoQAMGVk5eUgFRWmhELScoQBWOq4wwIrWtvaWwEN2Dk4u7mCk3r7+QWER0XGJucmcqek6mmiCaLlloeUAzIN7B7ltHd29-YM2E-ADIKgYCMDx7PsSUCg6HRuFAkciSbgoDpILR9ODYbC4QRkeKGHKcTD-eJNNzsdh9AYQJEo2ZcF6Kb77eCIZSQWBwIA&css=Q

**Comment by alfonsogarciacaro** [comment]
Ah, ok. I think we need to update the Fable.Core used by the REPL to the version including the AttachMembers attribute.

**Comment by alfonsogarciacaro** [comment]
I've updated the REPL, you should be able to use `AttachMembers` now (please do a hard reload if you've issues). Also, your sample made me realize there was a bug with static constructor in classes decorated with `AttachMembers`. This should be fixed in Fable 3.1.5.

> Please note that combining `Erase` and `AttachMembers` is not expected. If `Controller` is coming from a external source, you can use the `Global` or `Import` attributes instead (in that case you don't need the `AttachMembers` attribute because members are already considered to be attached for external types).
